src/library/Uno.Material/Controls/BottomNavigationBarItem.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Uno.Material.Controls
+{
+	public partial class BottomNavigationBarItem : ToggleButton
+	{
+		public string Data
```

Comment:
Why `string Data` and not `object Content`?

Reply:
I actually wanted to use Geometry instead of anything else but string worked and because it implicitly casts to a geometry properly.